packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -2996,16 +3038,6 @@ describe('plugin-meetings', () => {
           assert.isOk(errorThrown);
         });
 
-        it('should pass the forceTurnDiscovery parameter to doTurnDiscovery()', async () => {
-          meeting.meetingState = 'ACTIVE';
-
-          // call addMedia() with forceTurnDiscovery=true
-          await meeting.addMedia({}, undefined, true);
-
-          // Check that the value was passed on to doTurnDiscovery()
-          assert.calledOnceWithExactly(meeting.roap.doTurnDiscovery, meeting, false, true);
-        });
-
```

Comment:
Do we test this elsewhere? Why is this test being removed?

Reply:
It's removed, because addMedia() doesn't have the `forceTurnDiscovery` argument anymore. This argument exists now only in addMediaInternal() which is now a non-public function, so I didn't want to call it directly from the tests. I thought the logic of forcing TURN discovery through addMediaInternal() was already tested in one of joinWithMedia() tests, but turns it those tests were stubbing addMediaInternal(), so I've now modified one of them so that it checks that TURN discovery is forced by addMediaInternal() when it's called from joinWithMedia().

Reply:
Thanks for the explanation, @marcin-bazyl